openedxscorm/scormxblock.py

```diff
@@ -24,6 +28,10 @@ def _(text):
 
 logger = logging.getLogger(__name__)
 
+# importing directly from settings.XBLOCK_SETTINGS doesn't work here... doesn't have vals from ENV TOKENS yet
+scorm_settings = settings.ENV_TOKENS["XBLOCK_SETTINGS"].get("ScormXBlock", {})
+SCORM_MEDIA_BASE_URL = scorm_settings.get("SCORM_MEDIA_BASE_URL", "/scorm")
+
```

Comment:
If the variable is required in only one method, we can import `settings.XBLOCK_SETTINGS` in the method?

Reply:
considering it as a constant, I think it should be available at the top.